association_backend/association/api/builder.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import field, make_dataclass
```

Comment:
reminder for me: Ideally this should live in a shared codebase, I am thinking of releasing it as a lib